crates/driver/src/tests/setup/mod.rs

```diff
@@ -1224,6 +1224,41 @@ impl<'a> SolveOk<'a> {
         assert!(self.solutions().is_empty());
     }
 
+    /// Check that the solution contains the expected JIT orders.
+    pub fn jit_orders(self, jit_orders: &[JitOrder]) -> Self {
+        let solution = self.solution();
+        assert!(solution.get("orders").is_some());
+        let trades = serde_json::from_value::<HashMap<String, serde_json::Value>>(
+            solution.get("orders").unwrap().clone(),
+        )
+        .unwrap();
+
+        // Since JIT orders don't have UID at creation time, we need to search for
+        // matching token pair
+        for expected in jit_orders.iter() {
+            for trade in trades.values() {
+                let u256 = |value: &serde_json::Value| {
+                    eth::U256::from_dec_str(value.as_str().unwrap()).unwrap()
+                };
+                let sell_token = trade.get("sellToken").unwrap().to_string();
+                let buy_token = trade.get("buyToken").unwrap().to_string();
+
+                if sell_token == expected.order.sell_token && buy_token == expected.order.buy_token
+                {
+                    assert!(is_approximately_equal(
```

Comment:
Is `is_approximately_equal` actually necessary here?
Don't we know for sure which values to expect exactly?